plugins/README.md

```diff
@@ -0,0 +1,5 @@
+# Geometry plugins
```

Comment:
I have a README almost ready, was working for the current version. Just need to adapt it. Can you change the base branch to something like `2.0` instead of `master`? That way I can contribute as well.